game/theater/theatergroundobject.py

```diff
@@ -481,18 +502,77 @@ def __init__(
             dcs_identifier="AA",
             sea_object=False,
         )
-        # Set by the SAM unit generator if the generated group is compatible
-        # with Skynet.
-        self.skynet_capable = False
+        self.aa_ranges: List[AirDefenseRange] = [aa_range]
+
+    # Returns the name of one single group
+    # It also changes the group name to a skynet readable
+    # If the skynet conditions are true (skynet_capability_for_range)
+    def single_group_name(self, group_index: int) -> str:
+
+        if self.is_skynet_capable(group_index):
```

Comment:
I don't think this check is actually necessary: https://github.com/dcs-liberation/dcs_liberation/issues/405

aiui skynet will filter out invalid sites on its own.